hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicRootedOzoneFileSystem.java

```diff
@@ -768,13 +768,19 @@ private boolean deleteInSpan(Path f, boolean recursive) throws IOException {
 
   private boolean deleteBucket(Path f, boolean recursive, OFSPath ofsPath)
       throws IOException {
+    OzoneBucket bucket;
+    try {
+      bucket = adapterImpl.getBucket(ofsPath, false);
+    } catch (Exception ex) {
```

Comment:
Can you add a log statement here signifying there was an error.
```
      LOG.error("Exception while getting bucket link information, " +
          "considered it as false", ex);
```

Reply:
Thank you @ArafatKhan2198 for the review, added the logging statement as suggested!

Reply:
Trying to delete non-existent bucket is not an error, so we should ignore `OMException` with `BUCKET_NOT_FOUND` or `VOLUME_NOT_FOUND` results silently.

Reply:
Right makes sense.

Reply:
Thank you @adoroszlai for mentioning this, please review the updated changes, thanks!